### PR TITLE
feat: list component added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
List component is added.
There are two types of lists.
1. Simple list.
2. stacked list

### CODE REVIEW

File: componentLibrary.css [hunk 1]
```diff
@@ -593,3 +593,29 @@
     align-items: center;
     color: white;
 }
+
+/* list css starts from here  */
+
+.list {
+    border: 1px solid var(--primary-color);
+    /* margin: 0; */
```
Comment (fahd23) [inline componentLibrary.css:601]: you can remove commented code

**Review by fahd23 — APPROVED** [review]
Code looks good

**Review by SaurabhChawda — APPROVED** [review]
good to merge.

File: componentLibrary.css (hunk 1 above)
Comment (ShreyashMahajan) [reply componentLibrary.css:601]: thank you i will do that

**Pull request merged**